### ISSUE TITLE
[REQUEST] x86 32bit and arm64 builds

### ISSUE THREAD

**Issue by reukiodo** [aristocratos/btop4win#35]
**Is your feature request related to a problem? Please describe.**
It would be nice to run btop on 32bit Win10 systems as well as Arm 32/64 Windows systems

**Describe the solution you'd like**
The Releases could have x86 32/64 and arm 32/64 builds

**Describe alternatives you've considered**
Well, arm/64 has x86 emulator, so it can run btop right now... but using more battery to do so.

**Additional context**
I have so many old systems, some still running x86/32 Windows 10, some are arm/64 running Windows 11, and yes, even an older arm/32 running Windows 10.

**Comment by rs38** [comment]
btw: ARM emulator does a surprisingly good job with btop4win x64

**Comment by reukiodo** [comment]
Yes, that's acknowledged in the original request above. It is ok, but doesn't run as low power as native.

**Comment by rs38** [comment]
![Image](https://github.com/user-attachments/assets/0837bd8d-4477-49d5-b6b3-89b06402d02c)

cloned the project and compiled 

`msbuild btop4win.sln /p:Configuration=Release /p:Platform=ARM64`

which creates a 3MB binary.

**Comment by aristocratos** [comment]
@reukiodo  @rs38 
I'm likely (at some point) gonna try to merge the windows specific code back in to the main btop++ repository (https://github.com/aristocratos/btop) since btop4win is missing alot of needed fixes. (And unify how gpu information is shown, since it differs between the two.)

Shouldn't be a problem to also include win/arm64 binaries in the releases, x86 I'm not that sure about, but maybe.
